js/src/elements/form-behavior.ts

```diff
@@ -700,16 +703,17 @@ export function navigateAfterSubmit(href: string | null, navigationStep: Navigat
 					window.history.pushState(state, document.title, window.location.pathname + search + href); // prevents scrolling to spot until reload
 				}
 				location.reload();
+				return true;
 			}
-			break;
+			return false;
 		case 'replace':
 			if (href) {
 				location.replace(href);
 			}
-			break;
+			return true;
```

Comment:
false when there's no href?

Reply:
Updated! I thought this would always reload the page with or without an href, but have learned otherwise